modules/2_nbde/variables.tf

```diff
@@ -182,13 +182,17 @@ variable "tang" {
     # optional fixed IPs
     # fixed_ips = []
     # optional data volumes to master nodes
-    # data_volume_size = 100 #Default volume size (in GB) to be attached to the master nodes.
-    # data_volume_count = 0 #Number of volumes to be attached to each master node.
+    data_volume_size  = 10 #Default volume size (in GB) to be attached to the Tang servers.
+    data_volume_count = 1  #Number of volumes to be attached to each Tang server.
   }
   validation {
     condition     = lookup(var.tang, "count", 3) == 3
     error_message = "The tang.count value must be 3."
   }
+  validation {
```

Comment:
Push this down to the main variables.tf, and leave it in the module too.

Reply:
As soon as I start the deployment it throws the error due to the validation added in 2_nbde/variables.tf
```
# terraform apply -var-file=var.tfvars --auto-approve |  tee tang-0903-1
module.bastion[0].data.openstack_networking_network_v2.network: Reading...
module.bastion[0].data.openstack_compute_flavor_v2.bastion: Reading...
module.bastion[0].data.openstack_networking_network_v2.network: Read complete after 1s [id=05d0f439-963b-49bd-ba09-4eec4113a8bc]
module.bastion[0].data.openstack_compute_flavor_v2.bastion: Read complete after 2s [id=4fddff94-a364-4e64-9916-ff0c11356afc]
╷
│ Error: Invalid value for variable
│
│   on main.tf line 104, in module "nbde":
│  104:   tang                            = var.tang
│     ├────────────────
│     │ var.tang is object with 5 attributes
│
│ The tang.data_volume_count must be 1.
│
│ This was checked by the validation rule at modules/2_nbde/variables.tf:192,3-13.
```